Lib/test/bisect_cmd.py

```diff
@@ -51,6 +51,12 @@ def python_cmd():
     cmd = [sys.executable]
     cmd.extend(subprocess._args_from_interpreter_flags())
     cmd.extend(subprocess._optim_args_from_interpreter_flags())
+    for key, value in sys._xoptions.items():
```

Comment:
I don't like this code. It produces duplicated ``-X dev`` option. It calls 2 subprocess functions and then add more options.

IMO what can be done is to take the original command line and replace ``-m test.bisect_cmd`` with ``-m test``. Can you try to start from ``sys.orig_argv`` instead? I added it to Python 3.10. But please support the uncommon case when the script is run directly. Example: ``./python Lib/test/bisect_cmd.py test_os``. Maybe try to look to an argument equal to ``__file__`` or ending with ``os.path.basename(__file__)`` and replace it with ``-m test``?